test-projects/nest-10/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "nest-10",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "build": "nest build --type-check",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:e2e": "jest --config ./test/jest-e2e.json"
+  },
+  "dependencies": {
+    "@nestjs/common": "^10.0.0",
+    "@nestjs/core": "^10.0.0",
+    "@nestjs/platform-express": "^10.0.0",
+    "@swc/cli": "0.1.62",
+    "@swc/core": "^1.3.62",
+    "reflect-metadata": "^0.1.13",
+    "rxjs": "^7.8.1",
+    "@ts-rest/core": "file:../../dist/libs/ts-rest/core",
```

Comment:
ah, so we install it locally via file system/symlink? 
Do we forsee any caching issues with this?

Reply:
Caching issues? wdym sorry?

As I'm running `pnpm exec nx run-many --target=build --projects="ts-rest*" `

before this it should always have the latest packages :D

Reply:
Nvm, confused PNPM with npm 😅